File: WebNaim/PiDevF2/PiDev/src/PidevBundle/Entity/Programme.php
<?php

namespace PidevBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Programme
 *
 * @ORM\Table(name="programme")
 * @ORM\Entity
 */
class Programme
{
    /**
     * @var integer
     *
     * @ORM\Column(name="IdProgramme", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $idprogramme;

    /**
     * @var integer
     *
     * @ORM\ManyToOne(targetEntity="Activite")
     * @ORM\JoinColumn(name="IdActivite",referencedColumnName="IdActivite", onDelete="CASCADE")
     *
     */

    private $idactivite;

    /**
     * @return int
     */
    public function getIdprogramme()
    {
        return $this->idprogramme;
    }

    /**
     * @param int $idprogramme
     */


    /**
     * @return int
     */
    public function getIdactivite()
    {
        return $this->idactivite;
    }

    /**
     * @param int $idactivite
     */
    public function setIdactivite($idactivite)
    {
        $this->idactivite = $idactivite;
    }

    /**
     * @return string
     */
    public function getCours()
    {
        return $this->cours;
    }

    /**
     * @param string $cours
     */
    public function setCours($cours)
    {
        $this->cours = $cours;
    }

    /**
     * @return string
     */
    public function getDebut()
    {
        return $this->debut;
    }

    /**
     * @param string $debut
     */
    public function setDebut($debut)
    {
        $this->debut = $debut;
    }

    /**
     * @return string
     */
    public function getFin()
    {
        return $this->fin;
    }

    /**
     * @param string $fin
     */
    public function setFin($fin)
    {
        $this->fin = $fin;
    }

    /**
     * @var string
     *
     * @ORM\Column(name="Cours", type="text", nullable=false)
     */
    private $cours;

    /**
     * @var string
     *
     * @ORM\Column(name="Debut", type="string", length=30, nullable=false)
     */
    private $debut;

    /**
     * @var string
     *
     * @ORM\Column(name="Fin", type="string", length=30, nullable=false)
     */
    private $fin;










}

File: IntegrationFinale/User/src/entity/Programme.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package entity;

/**
 *
 * @author ASUS
 */
public class Programme {
     private int IdProgramme;
     private String Intitule;

     private String Cours;
     private String Debut;
     private String Fin;

    public Programme() {
    }
    
      public Programme( String Intitule,String Cours, String Debut, String Fin) {
        this.Intitule=Intitule;
          this.Cours = Cours;
        this.Debut = Debut;
        this.Fin = Fin;
    }

    public Programme(int IdProgramme,String Intitule, String Cours, String Debut, String Fin) {
               this.Intitule=Intitule;

        this.IdProgramme = IdProgramme;
        this.Cours = Cours;
        this.Debut = Debut;
        this.Fin = Fin;
    }

    public String getIntitule() {
        return Intitule;
    }

    public void setIntitule(String Intitule) {
        this.Intitule = Intitule;
    }

 

    public String getCours() {
        return Cours;
    }

    public String getDebut() {
        return Debut;
    }

    public String getFin() {
        return Fin;
    }

    public int getIdProgramme() {
        return IdProgramme;
    }

    public void setCours(String Cours) {
        this.Cours = Cours;
    }

    public void setDebut(String Debut) {
        this.Debut = Debut;
    }

    public void setFin(String Fin) {
        this.Fin = Fin;
    }

    public void setIdProgramme(int IdProgramme) {
        this.IdProgramme = IdProgramme;
    }

    @Override
    public String toString() {
        return "Programme{" + "IdProgramme=" + IdProgramme + ", Cours=" + Cours + ", Debut=" + Debut + ", Fin=" + Fin + '}';
    }
     

          
}
